clients/cli/src/test/java/org/apache/gravitino/cli/integration/test/TableFormatOutputIT.java

```diff
@@ -152,11 +154,11 @@ public void testCatalogDetailsCommand() {
     // Get the output and verify it
     String output = new String(outputStream.toByteArray(), StandardCharsets.UTF_8).trim();
     assertEquals(
-        "+----------+------------+-----------------+---------+\n"
-            + "| catalog  | type       | provider        | comment |\n"
-            + "+----------+------------+-----------------+---------+\n"
-            + "| postgres | RELATIONAL | jdbc-postgresql | null    |\n"
-            + "+----------+------------+-----------------+---------+",
+        "+----------+------------+-----------------+-------------------+\n"
+            + "| catalog  | type       | provider        | comment           |\n"
+            + "+----------+------------+-----------------+-------------------+\n"
+            + "| postgres | RELATIONAL | jdbc-postgresql | catalog，用于测试 |\n"
+            + "+----------+------------+-----------------+-------------------+",
```

Comment:
Don't modify original test case `testCatalogDetailsCommand()`

Please add a new test `postgres2` case to standalone test full corner character
```
testCatalogDetailsCommandFullCornerCharacter() {
    // Create a byte array output stream to capture the output of the command
    ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
    PrintStream originalOut = System.out;
    System.setOut(new PrintStream(outputStream));
    String[] args = {
      "catalog",
      "details",
      commandArg(GravitinoOptions.METALAKE),
      "my_metalake",
      commandArg(GravitinoOptions.NAME),
      "postgres2",
      commandArg(GravitinoOptions.OUTPUT),
      "table",
      commandArg(GravitinoOptions.URL),
      gravitinoUrl
    };
    Main.main(args);
...
```
This way, We can test full-corner characters and half-corner characters. :-)